doc/release/1.0.0-notes.rst

```diff
@@ -75,6 +75,14 @@ has been changed to True.
 
 `scipy.special.gammaln` does not accept complex arguments anymore.
 
+The cross-class properties of `scipy.signal.lti` systems have been removed.
+The following properties/setters have been removed:
+
+Name - (accessing/setting has been removed) - (setting has been removed)
+* StateSpace - (`num`, `den`, `gain`) - (`zeros`, `poles`)
```

Comment:
for this to render correctly, need a blank line above the list and double backticks around the names